src/lib/analysers/convert_audio/main.py

```diff
@@ -0,0 +1,29 @@
+from lib.common.analyser import Analyser
+from subprocess import call, STDOUT
+from pathlib import Path
+import os
+
+
+class ConvertAudioAnalyser(Analyser):
+    def analyse_element(self, element, config):
+
+        src = element["src"]
+        dest = element["dest"]
+        key = element["id"]
+        input_ext = config["input_ext"]
+        output_ext = config["output_ext"]
+        media = list(Path(src).rglob(f"*.{input_ext}"))
+
+        if len(media) is not 1:
+            raise Error(
```

Comment:
This should be `Exception`. `Error` is not defined in Python's core library.

Reply:
Fixed in commit 35cd5ce